core/sail/shacl/src/main/java/org/eclipse/rdf4j/sail/shacl/AST/package-info.java

```diff
@@ -1,8 +1,7 @@
-@Deprecated
 @InternalUseOnly
 /**
- * @deprecated since 3.0. This feature is for internal use only: its existence, signature or behavior may change without
- *             warning from one release to the next.
+ * @apiNote This feature is for internal use only: its existence, signature or behavior may change without warning from
+ *          one release to the next.
  */
 package org.eclipse.rdf4j.sail.shacl.AST;
 
```

Comment:
Could you skip the SHACL code, since I've rewritten the AST this might cause conflicts when merging to develop. Also, I've got you covered here: https://github.com/eclipse/rdf4j/pull/2773/files#diff-53faeded2adb71996f61cdbcacbe239464fbb3569deda113a3402412d01ba921

Reply:
Ah good point, will do